### PR TITLE
Make punctuation marks consistent for singular & plural strings

### LINKED ISSUE #3377: Translation: Differences in strings between singular / plural: ending / not ending on period
## Description
Singular ending on period, plural not here:

###count### image provided by this source. **<>** ###count### images provided by this source
###count### audio file with the selected tag. **<>** ###count### audio files with the selected tag
###count### image with the selected tag. **<>** ###count### images with the selected tag

compare:
Menu. ###count### filter applied

### PR DESCRIPTION
**Pull request by NekoJonez**
## Fixes
Fixes #3377 by @Presskopp 

## Description
This would solve https://github.com/WordPress/openverse/issues/3377. Small simple fix.

### CODE REVIEW

**Review by dhruvkb — APPROVED** [review]
I can't help being a pedantic PITA but I would prefer if we went the other way and didn't have the full-stop in the first half rather than adding them to the second.

But the other pedant in me like the added consistency so 👍!

**Review by obulat — CHANGES_REQUESTED** [review]
@dhruvkb has shown some examples in the issue of similar phrases that do not use periods. Let's remove them from both variants, instead of adding, @NekoJonez ?

File: frontend/src/locales/scripts/en.json5 [hunk 1]
```diff
@@ -804,24 +804,24 @@
       source: {
         audio: {
           zero: "No audio files provided by this source",
-          count: "{count} audio file provided by this source.|{count} audio files provided by this source",
+          count: "{count} audio file provided by this source.|{count} audio files provided by this source.",
```
Comment (obulat) [inline frontend/src/locales/scripts/en.json5:807]:
```suggestion
          count: "{count} audio file provided by this source|{count} audio files provided by this source",
```

File: frontend/src/locales/scripts/en.json5 [hunk 2]
```diff
@@ -804,24 +804,24 @@
...
           countMore: "Over {count} audio files provided by this source",
         },
         image: {
           zero: "No images provided by this source",
-          count: "{count} image provided by this source.|{count} images provided by this source",
+          count: "{count} image provided by this source.|{count} images provided by this source.",
```
Comment (obulat) [inline frontend/src/locales/scripts/en.json5:812]:
```suggestion
          count: "{count} image provided by this source|{count} images provided by this source",
```

File: frontend/src/locales/scripts/en.json5 [hunk 3]
```diff
@@ -804,24 +804,24 @@
...
           countMore: "Over {count} images provided by this source",
         },
       },
       tag: {
         audio: {
           zero: "No audio files with the selected tag",
-          count: "{count} audio file with the selected tag.|{count} audio files with the selected tag",
+          count: "{count} audio file with the selected tag.|{count} audio files with the selected tag.",
```
Comment (obulat) [inline frontend/src/locales/scripts/en.json5:819]:
```suggestion
          count: "{count} audio file with the selected tag|{count} audio files with the selected tag",
```

File: frontend/src/locales/scripts/en.json5 [hunk 4]
```diff
@@ -804,24 +804,24 @@
...
           countMore: "Over {count} audio files with the selected tag",
         },
         image: {
           zero: "No images with the selected tag",
-          count: "{count} image with the selected tag.|{count} images with the selected tag",
+          count: "{count} image with the selected tag.|{count} images with the selected tag.",
```
Comment (obulat) [inline frontend/src/locales/scripts/en.json5:824]:
```suggestion
          count: "{count} image with the selected tag|{count} images with the selected tag",
```

**Review by obulat — APPROVED** [review]

**Review by dhruvkb — APPROVED** [review]

**Pull request merged**